src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,27 @@ title = "Releases"
 weight = 30
 +++
 
+## v0.8.0
+
+* Added support for connecting clusters that use the OVN (ovn-kubernetes) CNI plugin.
+* The active Gateway now performs periodic health checks on the connections to remote clusters,updates the Gateway
+  connection status, and adds latency statistics.
+* Gateways now export the following connection metrics which can be used with Prometheus. These are currently only supported
+  for the Libreswan cable driver.
+  * The count of bytes transmitted and received between Gateways.
+  * The number of connections between Gateways and their corresponding status.
+  * The timestamp of the last successful connection established between Gateways.
+  * The RTT latency between Gateways.
+* The Libreswan cable driver is now the default.
+* The strongSwan cable driver is deprecated and will be removed in a future release.
+* Lighthouse now prefers a Service in the local cluster before choosing a remote cluster round-robin.
+* Lighthouse has fully migrated to use the new [Kubernetes Multicluster Service CRDs](https://github.com/kubernetes-sigs/mcs-api/)
```

Comment:
proposed API (instead of "new CRDs)?

Reply:
Is it still just proposed? That makes it seem like it's unstable and could change again or be replaced - do we want to convey that?

Reply:
IIUC, it's agreed but still needs to go through the Kubertenes graduation criteria. @skitt had a similar comment on my PR here: https://github.com/submariner-io/submariner-website/pull/383

Whatever we choose, let's try to be consistent.

Reply:
ok I'll change it.